### PR TITLE
feat: storybook integration

### PR DESCRIPTION
**Pull request by manucorporat**

### CODE REVIEW

File: starters/features/storybook/.storybook/main.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { StorybookConfig } from 'storybook-framework-qwik';
+
+const config: StorybookConfig = {
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
+    '@storybook/addon-a11y',
+    '@storybook/addon-coverage',
```
Comment (shairez) [inline starters/features/storybook/.storybook/main.ts:9]: same here, not required if not writing tests

File: starters/features/storybook/.storybook/main.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { StorybookConfig } from 'storybook-framework-qwik';
+
+const config: StorybookConfig = {
+  addons: [
+    '@storybook/addon-links',
+    '@storybook/addon-essentials',
+    '@storybook/addon-interactions',
+    '@storybook/addon-a11y',
```
[the remark this answers is not in the document]
Comment (manucorporat) [reply starters/features/storybook/.storybook/main.ts:8]: let's remove them then, it should be easy for storybook developers to add later, no?

[the remark this answers is not in the document]
Comment (manucorporat) [reply starters/features/storybook/.storybook/main.ts:8]: which addons should be included by default?

[the remark this answers is not in the document]
Comment (shairez) [reply starters/features/storybook/.storybook/main.ts]: yes

I believe these addons - 
'@storybook/addon-links',
'@storybook/addon-essentials'

But in terms of packages, all the rest are important (like vite etc)

**Pull request merged**